roles/download/defaults/main.yml

```diff
@@ -120,114 +93,97 @@ kubeadm_binary_checksum: "{{ kubeadm_checksums[kubeadm_version] }}"
+dashboard_image_tag: "{{ dashboard_version }}"
+dnsautoscaler_image_repo: "gcr.io/google_containers/cluster-proportional-autoscaler-{{ image_arch }}"
+dnsautoscaler_image_tag: "{{ dnsautoscaler_version }}"
+dnsmasq_image_repo: "andyshinn/dnsmasq"
+dnsmasq_image_tag: "{{ dnsmasq_version }}"
 dnsmasq_nanny_image_repo: "gcr.io/google_containers/k8s-dns-dnsmasq-nanny-{{ image_arch }}"
 dnsmasq_nanny_image_tag: "{{ kubedns_version }}"
 dnsmasq_sidecar_image_repo: "gcr.io/google_containers/k8s-dns-sidecar-{{ image_arch }}"
 dnsmasq_sidecar_image_tag: "{{ kubedns_version }}"
-dnsmasqautoscaler_version: 1.1.2
 dnsmasqautoscaler_image_repo: "gcr.io/google_containers/cluster-proportional-autoscaler-{{ image_arch }}"
 dnsmasqautoscaler_image_tag: "{{ dnsmasqautoscaler_version }}"
-dnsautoscaler_version: 1.3.0
-dnsautoscaler_image_repo: "gcr.io/google_containers/cluster-proportional-autoscaler-{{ image_arch }}"
-dnsautoscaler_image_tag: "{{ dnsautoscaler_version }}"
-test_image_repo: busybox
-test_image_tag: latest
-busybox_image_repo: busybox
-busybox_image_tag: 1.29.2
-helm_version: "v2.11.0"
+etcd_image_repo: "quay.io/coreos/etcd"
+etcd_image_tag: "{{ etcd_version }}{%- if image_arch != 'amd64' -%}-{{ image_arch }}{%- endif -%}"
+flannel_cni_image_repo: "quay.io/coreos/flannel-cni"
+flannel_cni_image_tag: "{{ flannel_cni_version }}"
+flannel_image_repo: "quay.io/coreos/flannel"
+flannel_image_tag: "{{ flannel_version }}"
 helm_image_repo: "lachlanevenson/k8s-helm"
 helm_image_tag: "{{ helm_version }}"
-tiller_image_repo: "gcr.io/kubernetes-helm/tiller"
-tiller_image_tag: "{{ helm_version }}"
-
-registry_image_repo: "registry"
-registry_image_tag: "2.6"
-registry_proxy_image_repo: "gcr.io/google_containers/kube-registry-proxy"
-registry_proxy_image_tag: "0.4"
-metrics_server_version: "v0.3.1"
+ingress_nginx_controller_image_repo: "quay.io/kubernetes-ingress-controller/nginx-ingress-controller"
+ingress_nginx_controller_image_tag: "{{ ingress_nginx_controller_version }}"
+install_socat_image_repo: "xueshanf/install-socat"
+install_socat_image_tag: "{{ socat_image_version }}"
+kube_router_image_repo: "cloudnativelabs/kube-router"
+kube_router_image_tag: "{{ kube_router_version }}"
+kubedns_image_repo: "gcr.io/google_containers/k8s-dns-kube-dns-{{ image_arch }}"
+kubedns_image_tag: "{{ kubedns_version }}"
+local_volume_provisioner_image_repo: "quay.io/external_storage/local-volume-provisioner"
+local_volume_provisioner_image_tag: "{{ local_volume_provisioner_version }}"
 metrics_server_image_repo: "k8s.gcr.io/metrics-server-amd64"
 metrics_server_image_tag: "{{ metrics_server_version }}"
-local_volume_provisioner_image_repo: "quay.io/external_storage/local-volume-provisioner"
-local_volume_provisioner_image_tag: "v2.1.0"
-cephfs_provisioner_image_repo: "quay.io/external_storage/cephfs-provisioner"
-cephfs_provisioner_image_tag: "v2.1.0-k8s1.11"
-ingress_nginx_controller_image_repo: "quay.io/kubernetes-ingress-controller/nginx-ingress-controller"
```

Comment:
why many things are moved around ? it's quite hard to follow what is removed, modified vs just reorganized.

Reply:
Ordered alphabetically to make things easier to find in the list.